x/posts/internal/keeper/common_test.go

```diff
@@ -46,6 +46,11 @@ func testCodec() *codec.Codec {
 var testPostOwner, _ = sdk.AccAddressFromBech32("cosmos1y54exmx84cqtasvjnskf9f63djuuj68p7hqf47")
 var timeZone, _ = time.LoadLocation("UTC")
 var testPostCreationDate = time.Date(2020, 1, 1, 15, 15, 00, 000, timeZone)
+var medias = types.PostMedia{
```

Comment:
No need to have this either, we can simply put it directly inside the `testPost` initialization

```golang
var testPost = types.NewPost(
	//...
       types.PostMedias{types.PostMedia{...}}
)
```

Also, I would advice using a `NewPostMedia` function instead of the type constructor